src/WebDocs/wwwroot/app/functions/CreateData/parameters.json

```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "Name": "DataKey",
+    "Description": "The identification of a item",
+    "Types": [ "datakey" ],
+    "Optional": false
+  },
+  {
+    "Name": "Notify",
+    "Description": "You can choose if you want no notify the change to the observer",
+    "Types": [ "boolean" ],
+    "Optional": true,
+    "DefaultValue": "true"
+  }
```

Comment:
There is 2 more parameters here

Reply:
![image](https://github.com/spadrapo/docs/assets/2657156/27ef7b1d-752c-43b1-ab51-59af7d5ae38a)
 
 It supports more than 2 parameters as you can see in the code

Reply:
Your sample have 6 parameters